libs/common/src/helpers/date-validation.ts

```diff
@@ -214,3 +214,17 @@ export const applyStandardValidationAndParseDateInput = (inputtedDate: DayMonthY
     parsedDate,
   };
 };
+
+/**
+ * checks if provided value is a date
+ * @param date - provided date
+ * @returns if the provided date is an instance of Date
+ */
+export const isDate = (date: any): boolean => date instanceof Date && !Number.isNaN(date.getTime());
+
+/**
+ * checks if provided value is a date and is not invalid
+ * @param date - provided date
+ * @returns if the provided date is an instance of Date and is not invalid
+ */
+export const isValidDate = (date: any): boolean => date instanceof Date && !Number.isNaN(date.getTime());
```

Comment:
`isDate` and `isValidDate` are doing the same thing, can we just have `isValidDate`?

Reply:
I am not sure if there will be any issues with adding `Number.isNaN(date.getTime())` to the existing functionality so have added a second function where I know it is required

Reply:
I don't understand, they're both doing exactly the same thing, we don't need 2x instances of a function doing the same thing

Reply:
I need to recommit, forgot to remove something 

`export const isDate = (date: any): boolean => date instanceof Date;` isDate should only be this without the !Number.isNaN(date.getTime())

Reply:
Resolving this with a fresh comment